Gaijinizer patch/Map359.txt

```diff
@@ -17,10 +17,10 @@ became so important...#GT
 ヘカーテがいないからって油断した私らも私らだし
 エンプーサの言う通り、次はこうはいかないから
 > CONTEXT: Map359/events/10/pages/0/8/Dialogue/enemy
-Alright, \\N[1].
-
-We're weird because of the absence of Hecate,
-and as Empusa says, we can't do this next time.#GT
+It's okay, \\N[1].
+We let our guard down because Hecate wasn't
+there, just like Empuse said, it won't happen
+again.#Ron
 > END STRING
```

Comment:
Minor rewording here:
```
It's okay, \\N[1].
We let our guard down because Hecate wasn't
here. Just like Empuse said, it won't happen
again.
```